src/pages/Home.tsx

```diff
@@ -1,29 +1,44 @@
 import {
   Box,
-  Tiles,
-  Inset,
   ContentWithSidebar,
   Body,
+  Tiles,
+  AreaLoader,
+  Feedback,
+  Inset,
 } from "@buildo/bento-design-system";
 import RestaurantPreview from "../components/RestaurantPreview";
-import rest_detail from "../mock-data/rest_detail.json";
+import { useQuery } from "@tanstack/react-query";
+import { getRestaurantList } from "../apis/api";
+import { fromJsonToProp } from "../utils";
+import { FiltersParams } from "../models";
 
-function Home() {
-  const name = rest_detail.name;
-  const imageUrl = rest_detail.image_url;
-  const rating = rest_detail.rating;
-  const address = rest_detail.location.address1;
+function Home({ range }: FiltersParams) {
```

Comment:
I'm guessing you have this prop here because of my comment in my previous review. What I had in mind, though, was creating a custom hook `useGetRestaurantList` that wraps the `useQuery` usage, with a single `range` parameter.
Having a prop is useful when you need to pass data from outside the component, but here this doesn't make much sense.

Reply:
I'm quite confused, so: 
- where should i put/it is good practice to put the APIs calls? --> I'm thinking: eventually I'll need to use user-set params for requests. With these specific project requirements, everything could be in the Home component happily I guess, but what if the App component manage other pages that require APIs?
- "range" for now is a placeholder for all filters
- I got the "wrapper" issue, sounds right that the query should not be invoked with any kind of component refresh